config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml

```diff
@@ -23,6 +23,55 @@ presubmits:
         - "./hack/verify-all.sh"
   # conformance test against kubernetes master branch with `kind`, skipping
   # serial tests so it runs in ~20m
+  # IPv6 enabled variant
+  - name: pull-kind-conformance-parallel-ipv6
+    labels:
+      preset-service-account: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    always_run: false
+    optional: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190515-3350b59-master
```

Comment:
same job as below, but newer image, optional, run_always: false, `-ipv6` in the name, and the ipv6 envs added